config/routes.rb

```diff
@@ -9,9 +12,13 @@
   devise_for :users
```

Comment:
Great job adding routes to your devise routes, but you only need to do it one time! Kindly delete one of them...